src/main/java/uk/gov/hmcts/probate/service/template/pdf/PDFManagementService.java

```diff
@@ -39,11 +51,15 @@
     private final ObjectMapper objectMapper;
     private final HttpServletRequest httpServletRequest;
     private final PDFServiceConfiguration pdfServiceConfiguration;
+    private final FileSystemResourceService fileSystemResourceService;
+    
+    static final String SIGNATURE_DECRYPTION_IV = "P3oba73En3yp7ion";
```

Comment:
we may want to store this on vault too

Reply:
The Initialisation Vector doesn't need to be a secret value (I.e. stored in the Vault). From what I've read it's best to change the IV for each encryption which I'm documenting in confluence but that relies on updating the encrypted image each time with the new IV which isn't that practical in our use case.

From Wikipedia...
An initialization vector has different security requirements than a key, so the IV usually does not need to be secret.